src/views/Portal/utils/validation.ts

```diff
@@ -359,7 +363,7 @@ export const validatedemographicForm = (
         } else if (eventLocation.length > 100) {
           isValid = false
           updateErrorMessage(
-            "Inputted Location Length is too Long",
+            "Response has 100 char limit",
             "eventLocationErr",
             dispatchError
           )
```

Comment:
Can we refer to eventLocation in our error message on line 383 as "Event Location"?

Reply:
sorry for the code reference, I can't comment on code not added in this PR